libraries/Microsoft.Bot.Connector/Authentication/JwtTokenExtractor.cs

```diff
@@ -124,7 +132,13 @@ public async Task<ClaimsIdentity> GetIdentityAsync(string scheme, string paramet
 
         private bool HasAllowedIssuer(string jwtToken)
         {
+            if (!_tokenValidationParameters.ValidateIssuer)
+            {
```

Comment:
I don't understand this line. Has this ever worked? The logic has me quite confused...

Reply:
During tests I was trying to skip issuer validation and noticed that even if I set it to false it kept validating issuers. And it was because this line was not there. 

In production, generally ValidateIssuer is true, so it was always correct to validate the issuer.